jujugui/static/gui/src/app/app.js

```diff
@@ -407,6 +407,9 @@ YUI.add('juju-gui', function(Y) {
       */
       this.userPaths = new Map();
 
+      // Track anonymous mode.
+      this.anonymousMode = false;
```

Comment:
Is this the default state? Do we want it to be true by default? Whichever way, maybe add a little comment as to why it's one or the other?